# Generated for NaoQi method ALRedBallTracker.startTracker
# Start the tracker by Subscribing to Event redBallDetected from ALRedBallDetection module.

---


